ui/src/router.tsx

```diff
@@ -107,18 +113,6 @@ function additionalRoutes() {
   switch (process.env.REACT_APP_ADDITIONAL_ROUTES) {
     case "none":
       return [];
-
-    case "sd":
-      return [
-        {
-          path: "sdAdmin",
-          element: <SdAdmin />,
-        },
-        {
-          path: "studies",
-          element: <Studies />,
-        },
```

Comment:
This removes the SD admin pages from the routing, but the code for them still lives in this GH repo, right? Is the plan to move those into the VUMC repo at some point?